src/slider/slider.scss

```diff
@@ -0,0 +1,301 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../icon/index';
+
+$thumb-height: $iui-m;  // 16px
+$track-height: $iui-xs;  // 4px
+$tick-height: $iui-s+$iui-xxs;
+
+@mixin iui-slider-component-container {
+  display: inline-flex;
+
+  span.iui-left,
+  span.iui-right {
+    @include iui-font-family;
+    user-select: all;
+    margin-top: $iui-xxs;
+
+    .iui-icon {
+      @include iui-icons-default;
+      margin-top: 4px;
```

Comment:
Use variable

Reply:
or actually..Better apply flex to the container and `align-items: center` and this margin would be not needed anymore..

Reply:
align-items: center does not look nice when tick marks are displayed. I like the text/icons to be aligned with slider track.

Reply:
Oh yeah.. I didnt notice it before.. Its because span takes full height (including marks...).